File: challenges/codeeval/easy/062/b1nary.clj
(require '[clojure.string :as s])
(defn parse-number
  [s]
  (if (re-find #"^-?\d+\.?\d*$" s)
    (read-string s)))
(defn getMod 
  [a]
    (mod (parse-number (first a)) (parse-number (second a))))
(defn trim_s
  [text]
    (s/split text #","))
(with-open [rdr (clojure.java.io/reader (first *command-line-args*))]
  (doseq [line (remove empty? (line-seq rdr))]
    (println(getMod(trim_s line)))))

File: challenges/codeeval/easy/023/b1nary.java
import java.io.*;
public class Main {
	public static void main (String[] args) throws IOException {
		for (int x=1; x<=12; x++){
			System.out.print(x);
			for (int y=2; y<=12 ; y++){
				if (x*y >= 10){
					if (x*y >= 100){
						System.out.print(" "+x*y);
					}
					else{
						System.out.print("  "+x*y);
					}
				}
				else{
					System.out.print("   "+x*y);
				}
			}
			System.out.println();
		}
	}
}